lib/inspec/objects/describe.rb

```diff
@@ -57,8 +57,8 @@ def initialize
       @variables = []
     end
 
-    def add_test(its, matcher, expectation)
-      test = Inspec::Describe::Test.new(its, matcher, expectation, false)
+    def add_test(its, matcher, expectation, negated = false)
```

Comment:
+1 on doing this.
I notice in the existing unit tests that it's hinted that you can use `obj.add_test(:aba, 'cmp', 'cba').negate!` for a negated test, and this change would not be needed.
I would personally not have guessed/assumed that the method returned the added test and I could chain a method call on it. 
We're also processing a large number of tests with a random mixture of negated ones. It would be nicer to have a single method call that we pass the boolean to instead of needing to write a branch that does two different calls to `add_test`